source/slang/slang-ir-util.cpp

```diff
@@ -1338,6 +1351,22 @@ List<IRBlock*> collectBlocksInRegion(IRGlobalValueWithCode* func, IRLoop* loopIn
     return collectBlocksInRegion(dom, loopInst, &hasMultiLevelBreaks);
 }
 
+IRBlock* getBlock(IRInst* inst)
+{
+    if (!inst)
+        return nullptr;
+
+    do {
```

Comment:
We usually put braces in its own line.

Reply:
ok i can update it

Reply:
do you want me to have it like
do
{
...
}
while(inst);

Reply:
That is fine, although I think the existing format is
```
do
{
} while (inst);
```
We don't use do-whiles that much in the codebase, so might be easier to just use `while(inst){}` or
```
for(;;)
{
       if (!inst) break;
}
```